.env.example

```diff
@@ -9,3 +9,21 @@ REACT_APP_SUPPORT_URL=
 REACT_APP_SENTRY_DSN_URL=
 REACT_APP_SENTRY_ENV=development
 SENTRY_AUTH_TOKEN=
+
+## Chainbridge configuration
+### Ethereum 
+REACT_APP_ETHEREUM_CHAIN_ID="5"

```

Comment:
What is CHAIN ID?

Reply:
It is the Ethereum Chain Id. For each chain, it will vary.